bulk_query/query.go

```diff
@@ -347,7 +362,7 @@ loop:
 			if q.gradualWorkersIncrease && !q.isBurnIn {
 				if q.gradualWorkersMax <= 0 || q.workers <= q.gradualWorkersMax {
 					for i := 0; i < workersIncreaseStep; i++ {
-						//fmt.Printf("Adding worker %d\n", workers)
+						//log.Printf("Adding worker %d\n", workers)
```

Comment:
Should we delete this comment instead of editing it?

Reply:
oh yeah, probably. will do